joomla/cassiopeia
# Use the right font name "fira-sans" and not just "fira" for the scss file name

Pull Request for PR #71 .

### Summary of Changes

Change name of the scss file for the "Fira Sans (web)" font scheme so it follows the naming scheme of the other fonts schemes scss file, i.e. contains the name of the font, which is "Fira Sans" and not "Fira".

This is not needed for functionality, but it is needed as a good example for a consistent naming of font scheme files.

If we are not consistent in the core, how can we teach developers to be consistent?

### Testing Instructions

Check that the naming scheme for the fonts scheme scss file is consistent.

Check that compiling scss and using fonts scheme "Fira Sans" still works when this PR here has been applied.

### Expected result

Names of font scheme scss files ontain the name of the font in all cases.

### Actual result

The name of the font scheme file `fonts-web_fira.scss` does not follow the naming scheme because the name of the onts is not complete, the "-sans" is missing.
